controllers/front/StoresController.php

```diff
@@ -225,6 +225,9 @@ public function getTemplateVarStores()
                 ],
             ];
             $store['image'] = $imageRetriever->getImage(new Store($store['id_store']), $store['id_store']);
+            if (is_array($store['image']['legend'])) {
+                $store['image']['legend'] = $store['image']['legend'][$this->context->language->id];
```

Comment:
It's you who knows, I just said that in case I did not understand. 
In my mind, `isset($object->meta_title) ? $object->meta_title : $object->name,``was a string and not an array, but if you can tests it to be sure.
```php 
if (is_array($store['image']['legend'])) {
    $store['image']['legend'] = $store['image']['legend'][$this->context->language->id]; 
} else {
    $store['image']['legend'] = $store['image']['legend'];
}
```
? like i said, it's you who knows :smile: